RxSwift/Traits/Infallible/Infallible+Operators.swift

```diff
@@ -288,12 +301,29 @@ extension InfallibleType {
      Observable of Observable sequences and that at any point in time produces the elements of the most recent inner observable sequence that has been received.
      */
     public func flatMapLatest<Source: ObservableConvertibleType>(_ selector: @escaping (Element) -> Source)
-        -> Infallible<Source.Element> {
+        -> Observable<Source.Element> {
+        asObservable().flatMapLatest(selector)
+    }
+
+    /**
+     Projects each element of an infallible sequence into a new sequence of infallible sequences and then
+     transforms the infallible sequence of infallible sequences into an infallible sequence producing values only from the most recent infallible sequence.
+
+     It is a combination of `map` + `switchLatest` operator
```

Comment:
Given that `switchLatest` doesn't exist for `Infallible,` should this line be kept or not?